tools/donate-cpu-server.py

```diff
@@ -1585,7 +1585,7 @@ def server(server_address_port: int, packages: list, packageIndex: int, resultPa
 
         print_ts('packages_nodata: {}'.format(len(packages_nodata)))
 
-        print_ts('removing packages with no files to process'.format(len(packages_nodata)))
```

Comment:
Wasn't the intent to print the number of packages (i.e. `{}` missing)?

Reply:
No. It is just c&p error - this is just to indicate the operation is taking place. There are further log message which provide all the remaining details.